### PR TITLE
bump jclouds version to 1.9.0

### PR DESCRIPTION
**Pull request by andreaturli**
removed virtualbox references

### CODE REVIEW

File: blobstore-basics/src/main/java/org/jclouds/examples/blobstore/basics/MainApp.java [hunk 1]
```diff
@@ -116,11 +113,8 @@ public static void main(String[] args) throws IOException {
             S3Client api = context.unwrapApi(S3Client.class);
             object = api.headObject(containerName, blobName);
          } else if (apiMetadata instanceof SwiftApiMetadata) {
-            SwiftClient api = context.unwrapApi(SwiftClient.class);
-            object = api.getObjectInfo(containerName, blobName);
-         } else if (apiMetadata instanceof AzureBlobApiMetadata) {
-            RestContext<AzureBlobClient, ?> providerContext = context.unwrap();
-            object = providerContext.getApi().getBlobProperties(containerName, blobName);
+            SwiftApi api = context.unwrapApi(SwiftApi.class);
+            object = api.getObjectApi(containerName, blobName);
```
Comment (demobox) [inline blobstore-basics/src/main/java/org/jclouds/examples/blobstore/basics/MainApp.java]: Could you add Azure back? I.e.

```
} else if (apiMetadata instanceof AzureBlobApiMetadata) {
  AzureBlobClient api = context.unwrapApi(AzureBlobClient.class);
  object = api.getBlobProperties(containerName, blobName);
} else if (apiMetadata instanceof SwiftApiMetadata) {
  ...
```

File: compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java [hunk 2]
```diff
@@ -137,6 +135,9 @@ public static void main(String[] args) {
 
       ComputeService compute = initComputeService(provider, identity, credential);
 
+       for (Location location : compute.listAssignableLocations()) {
+           System.out.println(location);
+       }
```
Comment (demobox) [inline compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java]: [minor] Indenting?

Comment (andreaturli) [inline compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java]: ops, it shouldn't be part of this PR, sorry

File: blobstore-basics/src/main/java/org/jclouds/examples/blobstore/basics/MainApp.java [hunk 3]
```diff
@@ -116,11 +114,11 @@ public static void main(String[] args) throws IOException {
...
          } else if (apiMetadata instanceof SwiftApiMetadata) {
-            SwiftClient api = context.unwrapApi(SwiftClient.class);
-            object = api.getObjectInfo(containerName, blobName);
+            SwiftApi api = context.unwrapApi(SwiftApi.class);
+            object = api.getObjectApi(containerName, blobName);
          } else if (apiMetadata instanceof AzureBlobApiMetadata) {
-            RestContext<AzureBlobClient, ?> providerContext = context.unwrap();
-            object = providerContext.getApi().getBlobProperties(containerName, blobName);
+            AzureBlobClient api = context.unwrapApi(AzureBlobClient.class);
+            object = api.getBlobProperties(containerName, blobName);
```
Comment (demobox) [inline blobstore-basics/src/main/java/org/jclouds/examples/blobstore/basics/MainApp.java]: Thanks, @andreaturli!

File: compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java [hunk 4]
```diff
@@ -146,9 +146,9 @@ public static void main(String[] args) {
             // that tested to work with java, which tends to be Ubuntu or CentOS
             TemplateBuilder templateBuilder = compute.templateBuilder();
 
-            if (providerIsGCE)
-               templateBuilder.osFamily(OsFamily.CENTOS);
-            
+            if (providerIsGCE) {
+                templateBuilder.osFamily(OsFamily.CENTOS);
```
Comment (demobox) [inline compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java]: [minor] Indent?

File: compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java [hunk 5]
```diff
@@ -160,12 +160,18 @@ public static void main(String[] args) {
...
-            NodeMetadata node = getOnlyElement(compute.createNodesInGroup(groupName, 1, templateBuilder.build()));
+            templateBuilder.options(runScript(bootInstructions));
+
+            Template template = templateBuilder.build();
+
+            if (providerIsGCE) {
+                GoogleComputeEngineTemplateOptions templateOptions = template.getOptions().as(GoogleComputeEngineTemplateOptions.class);
+                templateOptions.overrideLoginUser("andrea");
+                templateOptions.authorizePublicKey("ssh-rsa " +
+                        "AAAAB3NzaC1yc2EAAAADAQABAAABAQDNnvPZ25wZK19grrbal6R5JP1hLRBUuNh86KxZVMAFkGd5ouVSgO9dFajHZ45Q4mbaTkdOiqf7otMLDzkaztEa7oLK7Jso0Y0LOi+nT4gf38rvbEF5mq069G9b9XqlNleaGnpLuTN54iEK8c4TWZxIJqgelHEHhjp7V1asmilBbpZDmwA5cTt9vGJIhqA/BptKH3folZKeAQjRa1ZRSwSQUnk9rBKn4PKSRDojVa9A9jKt4qboJh5q7ZjtE8z+665F/4TgLzElTXUA8+uUFGpuynMSmQEt301e18dXAl+vBr8fMiThcoVVbdVdqdjXsw75fMXPAgqhjrw8k3+0/4P9 andrea@bigmac.local");
```
Comment (demobox) [inline compute-basics/src/main/java/org/jclouds/examples/compute/basics/MainApp.java]: [minor] Indent?

**Comment by andreaturli** [comment]
@demobox could we merge that now? thanks!

**Comment by demobox** [comment]
> could we merge that now? thanks!

Yes, looks good to me! Please squash'n'merge...

**Comment by andreaturli** [comment]
Merged at [549fba2cfc0ef3afef749e5231cf58dfe17b1498](https://git-wip-us.apache.org/repos/asf?p=jclouds-examples.git;a=commit;h=549fba2cfc0ef3afef749e5231cf58dfe17b1498)
